test/DocumentFormat.OpenXml.Packaging.Tests/OpenXmlPackageTests.cs

```diff
@@ -301,5 +304,29 @@ public void TestOpenModel3DWrittenByPowerPoint_DashMime()
 
             Assert.Equal("model/gltf-binary", model3DReferenceRelationshipPart.ContentType);
         }
+
+        // When opening a workbook (SpreadsheetDocument.Open) with a missing calcChain part and default OpenSettings
```

Comment:
These comments do not feel necessary

Reply:
I'll leave them for now since I'll probably be back in this code again soon.